### PR TITLE
doc/rgw: improve diagram caption

### PR DESCRIPTION
**Pull request by zdover23**
Improve the syntax and semantics of the caption of the diagram "Replication of Object Data Between Zones" in doc/radosgw/multisite.rst.

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

**Comment by zdover23** [comment]
This PR is a sequel to https://github.com/ceph/ceph/pull/50276.

File: doc/radosgw/multisite.rst [hunk 1]
```diff
@@ -66,10 +66,13 @@ something like this:
 
 At the top of this diagram, we see two applications (also known as "clients").
 The application on the right is both writing and reading data from the Ceph
-Cluster, by means of the RADOS Gateway (RGW).
+Cluster, by means of the RADOS Gateway (RGW). The application on the left is
+only reading data from the Ceph Cluster, by means of an instance of RADOS
```
Comment (anthonyeleven) [inline doc/radosgw/multisite.rst:70]: I might italicize _reading_ to disambiguate the message.

**Review by anthonyeleven — APPROVED** [review]
Non-blocking suggestion.

File: doc/radosgw/multisite.rst (hunk 1 above)
Comment (zdover23) [reply doc/radosgw/multisite.rst:70]: I'll do that. I appreciate the benefit of your fresh eyes.

**Comment by zdover23** [comment]
https://github.com/ceph/ceph/pull/50331 - Quincy backport
https://github.com/ceph/ceph/pull/50332 - Pacific backport

**Pull request merged**